Google.Api.Generator.Tests/ProtoTests/Basic/Testing.Basic/Testing.Basic.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+
+    <!-- TODO: Version defaults to 1.0.0, edit as required -->
+    <Version>1.0.0</Version>
```

Comment:
Perhaps we should default to a non-GA version, so that if anyone just publishes without checking, they don't end up with a V1 in the wild?

Reply:
sgtm. I'll address this in a new PR.